book_test.go

```diff
@@ -69,3 +71,26 @@ func TestLoadBook(t *testing.T) {
 		}
 	}
 }
+
+func TestApplyOptions(t *testing.T) {
+	tests := []struct {
+		opts []Option
+		want interface{}
+	}{
+		{[]Option{}, url.QueryEscape},
+		{[]Option{Debug(true)}, url.QueryEscape},
+		{[]Option{Func("gtEnv", os.Getenv)}, url.QueryEscape},
+		{[]Option{Func("urlencode", os.Getenv)}, os.Getenv},
+	}
+	for _, tt := range tests {
+		bk := newBook()
+		if err := bk.applyOptions(tt.opts...); err != nil {
+			t.Fatal(err)
+		}
+
+		got := bk.Funcs["urlencode"]
+		if reflect.DeepEqual(got, tt.want) {
```

Comment:
The evaluation formula was incorrect.
```suggestion
		if !reflect.DeepEqual(got, tt.want) {
```

However, the function comparison does not work.

Reply:
Oh...

Reply:
Fixed.
https://github.com/k1LoW/runn/pull/102/commits/137378ddfd419fef5762ad3478d1d0c7ebb242d4